assets/xml/objects/object_bg.xml

```diff
@@ -19,44 +22,58 @@
-        <Limb Name="object_bg_Standardlimb_008F3C" Type="Standard" EnumName="OBJECT_BG_1_LIMB_14" Offset="0x8F3C" />
-        <Limb Name="object_bg_Standardlimb_008F48" Type="Standard" EnumName="OBJECT_BG_1_LIMB_15" Offset="0x8F48" />
-        <Limb Name="object_bg_Standardlimb_008F54" Type="Standard" EnumName="OBJECT_BG_1_LIMB_16" Offset="0x8F54" />
-        <Limb Name="object_bg_Standardlimb_008F60" Type="Standard" EnumName="OBJECT_BG_1_LIMB_17" Offset="0x8F60" />
-        <Skeleton Name="object_bg_Skel_008FC8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_BG_1_LIMB_NONE" LimbMax="OBJECT_BG_1_LIMB_MAX" EnumName="ObjectBg1Limb" Offset="0x8FC8" />
+
+        <!-- Treasure Chest Shop Gal Textures -->
+        <Texture Name="gTreasureChestShopGalTLUT" OutName="treasure_cheat_shop_gal_tlut" Format="rgba16" Width="16" Height="16" Offset="0x68D8" />
+        <Texture Name="gTreasureChestShopGalHairTex" OutName="treasure_cheat_shop_gal_hair" Format="ci8" Width="32" Height="32" Offset="0x6AD8" />
+        <Texture Name="gTreasureChestShopGalHair2Tex" OutName="treasure_cheat_shop_gal_hair2" Format="ci8" Width="8" Height="16" Offset="0x6ED8" />
+        <Texture Name="gTreasureChestShopGalEyeOpenUpTex" OutName="treasure_cheat_shop_gal_eye_open_up" Format="ci8" Width="32" Height="32" Offset="0x6F58" />
+        <Texture Name="gTreasureChestShopGalEyeHalfUpTex" OutName="treasure_cheat_shop_gal_eye_half_up" Format="ci8" Width="32" Height="32" Offset="0x7358" />
+        <Texture Name="gTreasureChestShopGalEyeClosedTex" OutName="treasure_cheat_shop_gal_eye_closed" Format="ci8" Width="32" Height="32" Offset="0x7758" />
+        <Texture Name="gTreasureChestShopGalEyeOpenDownTex" OutName="treasure_cheat_shop_gal_eye_open_down" Format="ci8" Width="32" Height="32" Offset="0x7B58" />
+        <Texture Name="gTreasureChestShopGalEyeHalfDownTex" OutName="treasure_cheat_shop_gal_eye_half_down" Format="ci8" Width="32" Height="32" Offset="0x7F58" />
+        <Texture Name="gTreasureChestShopGalMouthTex" OutName="treasure_cheat_shop_gal_mouth" Format="ci8" Width="32" Height="32" Offset="0x8358" />
+        <Texture Name="gTreasureChestShopGalEarAndBellyButtonTex" OutName="treasure_cheat_shop_gal_ear_and_belly_button" Format="ci8" Width="16" Height="16" Offset="0x8758" />
+        <Texture Name="gTreasureChestShopGalSkinTex" OutName="treasure_cheat_shop_gal_skin" Format="ci8" Width="16" Height="16" Offset="0x8858" />
+        <Texture Name="gTreasureChestShopGalClothesTex" OutName="treasure_cheat_shop_gal_clothes" Format="ci8" Width="32" Height="32" Offset="0x8958" />
+        <Texture Name="gTreasureChestShopGalFingersTex" OutName="treasure_cheat_shop_gal_fingers" Format="ci8" Width="16" Height="16" Offset="0x8D58" /> 
+        
+        <!-- Treasure Chest Shop Gal Limbs -->
+        <Limb Name="gTreasureChestShopGal_008E58" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_01" Offset="0x8E58" />
```

Comment:
For this, and others like it, could you put `Limb` in there somewhere to make it clear that this is a limb? This is information that was in the original, undocumented name that is now lost.